grammar/haxe.bnf

```diff
@@ -301,7 +301,7 @@ enumConstructorParameters ::= '(' parameterList ')' {pin=1}
 
 varDeclaration ::= (macroMember | declarationAttribute)* 'var' varDeclarationPartList ';'
 {pin=3 mixin="com.intellij.plugins.haxe.lang.psi.impl.HaxePsiFieldImpl" implements="com.intellij.plugins.haxe.lang.psi.HaxePsiField"}
-private varDeclarationPartList ::= varDeclarationPart (',' varDeclarationPart)*
+private varDeclarationPartList ::= varDeclarationPart
```

Comment:
Oh, no.  I see.  This is for class fields only.  (localVarDeclaration is for method-local variables.)